packages/angular_devkit/core/node/testing/index.ts

```diff
@@ -20,7 +20,9 @@ export class TempScopedNodeJsSyncHost extends virtualFs.ScopedHost<fs.Stats> {
   protected override _root: Path;
 
   constructor() {
-    const root = normalize(path.join(os.tmpdir(), `devkit-host-${+Date.now()}-${process.pid}`));
+    const root = normalize(
+      path.join(os.tmpdir(), `devkit-host-${+performance.now()}-${process.pid}`),
```

Comment:
Would `crypto.randomUUID()` work here and be more fundamentally safe / non-racy?

Reply:
That sounds like a better long term option